client/src/components/StaticPagesMCK/Donate.js

```diff
@@ -1,171 +1,84 @@
+import IconSpacerSVG from "components/StaticPages/assets/IconSpacerSVG";
+import Footer from "components/Layout/Footer";
 
-const useStyles = makeStyles(() => ({
-  outer: {
-    background: "#fff",
-  },
-  main: {
-    padding: "1.5rem 0;",
-    maxWidth: "1200px",
-    margin: "0 auto",
-    "@media only screen and (min-width: 75em)": {
-      padding: "1.5rem 2rem",
-    },
-  },
-  title: {
-    textTransform: "uppercase",
-    fontWeight: 500,
-    textAlign: "center",
-    background: "#FFF",
-    margin: 0,
-    padding: "32px 0",
-  },
-  btnOrange: {
-    color: "#fff",
-    fontSize: "18px",
-    background: "#e57109",
-    boxShadow: "0px 4px 4px rgba(0, 0, 0, 0.25)",
-    borderRadius: "6px",
-    padding: "8px 16px",
-    textDecoration: "none",
-    textTransform: "uppercase",
-  },
-  btnOutline: {
-    color: "#fff",
-    border: "1px solid #336699",
-    background: "#336699",
-    borderRadius: "6px",
-    padding: "8px 16px",
-    textDecoration: "none",
-    textTransform: "uppercase",
-  },
-  btnWhite: {
-    color: "#336699",
-    background: "#ffffff",
-    borderRadius: "6px",
-    padding: "8px 16px",
-    textDecoration: "none",
-    textTransform: "uppercase",
-  },
-  figure: {
-    margin: 0,
-    padding: 0,
-  },
-  icon: {
-    margin: "auto",
-  },
-  donate: {
-    padding: "32px",
-    margin: "32px 0 0 0",
-    borderRadius: "24px",
-    background: "#f0f0f0",
-    display: "flex",
-    flexDirection: "column",
-    flexWrap: "wrap",
-    "& $h2": {
-      flexBasis: "100%",
-      textAlign: "center",
-      fontWeight: "500",
-      fontSize: "32px",
-      marginTop: "15px",
-      marginBottom: "15px",
-    },
-    "& $btnOutline": {
-      margin: "20px auto 0 auto",
-    },
-    "& $a": {
-      color: "#fff",
-      "&:hover": {
-        backgroundColor: "#0A3865",
-      },
-    },
-    "& $p": {
-      marginBottom: "16px",
-    },
-  },
-  volunteerSection: {
-    padding: "32px",
-    margin: "32px 0 0 0",
-    borderRadius: "24px",
-    background: "#B6D8FB",
-    display: "flex",
-    flexDirection: "column",
-    flexWrap: "wrap",
-    "& $h2": {
-      flexBasis: "100%",
-      textAlign: "center",
-      fontWeight: "500",
-      fontSize: "32px",
-      marginTop: "10px",
-      marginBottom: "20px",
-    },
-    "& $btnOutline": {
-      color: "#fff",
-      margin: "20px auto 16px auto",
-    },
-    "& $a": {
-      "&:hover": {
-        backgroundColor: "#0A3865",
-      },
-    },
-    "& $p": {
-      marginTop: "15px",
-    },
-  },
-  cards: {
-    display: "flex",
-    flexWrap: "wrap",
-    margin: "0 0 32px 0",
-    justifyContent: "center",
-    "@media only screen and (min-width: 64em)": {
-      flexWrap: "nowrap",
-      padding: "32px",
-    },
-  },
-  signup: {
-    background: "rgba(229, 113, 9, .7)",
-  },
-  volunteer: {
-    background: "#e57109",
-  },
-}));
 const Donate = () => {
-  const classes = useStyles();
   // const { t } = useTranslation("donate");
```

Comment:
I think we can delete this commented out code 👌

Reply:
I believe Qiqi also removed this line in her client clean up pr. I can see conflicts coming